source/reverse-sync.txt

```diff
@@ -0,0 +1,190 @@
+.. _c2c-reverse-process:
+
+======================
+Reverse Sync Direction
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :class: singlecol
+   :depth: 1
+
+To reverse the direction of a sync operation, use the following
+procedure to commit your sync and call the ``reverse`` endpoint. 
```

Comment:
Could we link the cutover [docs](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/cutover-process/) here too?

Reply:
Done!